### ISSUE TITLE
Double vertical scrolling bar

### ISSUE THREAD

**Issue by atais** [braginteractive/MDLWP#6]
Hello.

I have experienced a small bug in the CSS.
![zrzut ekranu 2015-08-22 o 23 56 28](https://cloud.githubusercontent.com/assets/1538192/9426055/c58b0e88-4929-11e5-9709-1a9246c9fb89.png)

I have managed to solve it easily by adding:

``` javascript
  body{
    overflow-y: hidden;
  }
```

Overflow on the page ribbon did not fix it for me.

**Comment by braginteractive** [comment]
Do you see the double vertical scrollbar on our demo ribbon page? http://mdlwp.com/demo/ribbon-page/

**Comment by atais** [comment]
Hm, no it's ok in the demo. I will try to find what is causing the issue.

**Comment by atais** [comment]
It is doubled when you log in to Wordpress and the Wordpress bar is visible.

**Comment by braginteractive** [comment]
I think because they are both fixed.

**Comment by atais** [comment]
you can change some div's to have `overflow-y: hidden;`. I will do some fixes and PR when I finish my fork.

**Comment by leoradmagnegithub** [comment]
You might resolve your problem by hiding the Wordpress bar. You should add these lines of code on the top of style.min.css

**#av_toolbar_regdiv, #av_toolbar_iframe, .av_site, .av-antipixel {
    display: none !important;
}
body {top: -40px !important;}**

I've found them on this website: http://tognolo.org/2012/10/guida-eliminare-la-toolbar-di-altervista/

**Comment by leoradmagnegithub** [comment]
On my website only on Mozilla Firefox (lastest version) there's a double vertical scrollbar, even if I have hidden the Wordpress status bar.

**Comment by braginteractive** [comment]
@leoradmagnegithub does the demo have a double scrollbar?

**Comment by leoradmagnegithub** [comment]
No, it doesn't.

**Comment by leoradmagnegithub** [comment]
My website work perfectly on every browser, except for Mozilla Firefox (v.45.0.1).
This because there's a double vertical scroll bar. 
This bar isn't in all other browsers like Chrome, Opera and IE 11. 

In the demo, this double vertical scroll bar never appear.

**Comment by ghost** [comment]
My website work perfectly on every browser, except for Mozilla Firefox (v.45.0.1).
This because there's a double vertical scroll bar. 
This bar isn't in all other browsers like Chrome, Opera and IE 11. 

In the demo, this double vertical scroll bar never appear.

![cattura](https://cloud.githubusercontent.com/assets/17053051/14761193/32f4ceba-0959-11e6-9b01-5e13518ebb09.PNG)

**Comment by alexandre67fr** [comment]
The double bar appears when administrators are logged in.

This is due to the fact that WP adds 

``` css
html { margin-top: 32px !important; }
```

The first scroll bar appears for `html` tag, the second one for `div.mdl-layout`

A solution simple solution for modern browsers in `functions.php`:

``` php
// Bugfix for double admin bar

add_action( 'wp_head', function () {

  if ( ! is_admin_bar_showing() )
    return;

  echo '
    <style>
      html {
        overflow: hidden;
      }
      .mdl-layout {
        height: calc(100% - 32px);
      }
      @media screen and (max-width: 782px) {
        .mdl-layout {
          height: calc(100% - 46px);
        }
      }
    </style>
  ';    

}, 999);
```

**Comment by leoradmagnegithub** [comment]
It doesn't work.

**Comment by alexandre67fr** [comment]
@leoradmagnegithub my comment is a reply to @atais 's issue, which is double vertical bar on ribbon template, when logged in as administrator in _all_ browsers.

The issue that you've posted is most probably a different one.

**Comment by ghost** [comment]
Ah ok, sorry.
Have you got any idea about my issue?

**Comment by alexandre67fr** [comment]
No, sorry. I didn't see any issues with my website using the latest version of the theme in Firefox.
Is your site online? I could have a look, and see what might be the issue.

**Comment by leoradmagnegithub** [comment]
My website is http://blog.giallozafferano.it/leoradmagne/

**Comment by alexandre67fr** [comment]
You're using a plugin that adds a bar at the top of the page. This not an issue of the theme.

**Comment by leoradmagnegithub** [comment]
I am using: Akismet, SI Captcha ANTI SPAM, custom meta box, jetpack by Wordpress, Yoast SEO.
Which of those can cause the issue?

**Comment by alexandre67fr** [comment]
You need help of a developer with your website, this is not an issue if this theme.

**Comment by leoradprogrammer** [comment]
Yes, sorry for the loss of time. I had had a problem on my site.

(This is the new account of the "ghost" in the previous comments)
